pkg/utils/utils.go

```diff
@@ -50,6 +50,13 @@ func Max(x, y int) int {
 	return y
 }
 
+func MinMax(x int, y int) (int, int) {
```

Comment:
Not a very intuitive name, I had to look at the implementation to figure out what it does. How about `SortRange`?

Reply:
Done